src/Game.java

```diff
@@ -0,0 +1,67 @@
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.Map.Entry.comparingByValue;
+import static java.util.stream.Collectors.toMap;
+
+public class Game {
+
+     public void startGame(String requiredText) {
+        String strToGuess = createStringToGuess(requiredText.length());
+        Set<String> setForEnteredValues = new HashSet<String>();
+
+        while (strToGuess.contains("*")) {
+            Scanner scanner = new Scanner(System.in);
+            String userInput = scanner.nextLine().toLowerCase();
+
+            if (!isAlpha(userInput)) {
```

Comment:
Here is output of your program if the whole name is entered
LET'S PLAY THE GAME, TRY TO GUESS THE WORD(S)
tatsiana masiukevich
Please enter the correct symbol(s), only letters can be used in name
You guessed right!
try to guess now where *, your result is tatsiana masiukevich
Congrats! You guessed the word(s).

Try to make your game more user friendly